src/subcommands/wallet/mod.rs

```diff
@@ -242,9 +243,9 @@ impl<'a> WalletSubCommand<'a> {
                 let payload = AddressPayload::from_pubkey_hash(hash160.clone());
                 lock_hashes.push(Script::from(&payload).calc_script_hash());
             }
-            last_change_address.payload().clone()
+            (last_change_address.payload().clone(), path_map.get(&change_last))
         } else if let Some((ref from_address_payload, _)) = from_address_info_opt {
-            from_address_payload.clone()
+            (from_address_payload.clone(), None)
```

Comment:
Assuming the from address is always something we have a non-derived private key for (I think this is true), we can do `DerivationPath::empty()` instead of `None`.